src/Client.cpp

```diff
@@ -183,6 +210,11 @@ void Client::OnAfterCreated(CefRefPtr<CefBrowser> browser)
 
     CEF_REQUIRE_UI_THREAD();
 
+    if (!m_requestHandler->m_messageRouterBrowserSide) {
```

Comment:
Cannot this initialization go to `CefClient::OnContextInitialized`? Or even better: to the constructor?